src/main/java/net/starschema/clouddb/jdbc/Oauth2Bigquery.java

```diff
@@ -285,7 +287,8 @@ public static Bigquery authorizeViaService(
       throws GeneralSecurityException, IOException {
     GoogleCredentials credential =
         createServiceAccountCredential(
-            serviceaccountemail, keypath, password, jsonAuthContents, false);
+                serviceaccountemail, keypath, password, jsonAuthContents, false)
+            .createScoped(GenerateScopes(false));
```

Comment:
nit: probably don't need to reset scopes here since `createServiceAccountCredential` handles that.